packages/daimo-expo-passkeys/src/index.ts

```diff
@@ -0,0 +1,105 @@
+import { decode as atob, encode as btoa } from "base-64";
+import { Platform } from "react-native";
+
+import ExpoPasskeysModule from "./ExpoPasskeysModule";
+import {
+  CreateRequest,
+  CreateResult,
+  SignRequest,
+  SignResult,
+  toAndroidCreateRequest,
+  toAndroidSignRequest,
+  toBase64,
+} from "./utils";
+
+export { CreateRequest, CreateResult, SignRequest, SignResult };
+
+/**
+ * Create a new passkey.
+ * All parameters are either strings or in regular base64 encoding.
+ *
+ * @param request.domain The domain to create the passkey for.
+ * @param request.challengeB64 The challenge to use for creation request.
+ * @param request.accountName The account name to attach the passkey to.
+ * @return result.rawClientDataJSONB64 The raw client data JSON.
```

Comment:
Yep. That's accurate. Most of the confusion from mixing the types of base 64 encoding is due iOS deciding to stick to base 64 rather than base 64 URL.